content/zh/docs/installing-on-linux/introduction/multioverview.md

```diff
@@ -358,6 +346,5 @@ kubectl completion bash >/etc/bash_completion.d/kubectl
 
 详细信息[见此](https://kubernetes.io/docs/tasks/tools/install-kubectl/#enabling-shell-autocompletion)。
 
-## 示例
-
-<script src="https://asciinema.org/a/364501.js" id="asciicast-364501" async></script>
```

Comment:
Do not change this demo. The ZH-CN version and the English version are supposed to be different. This part is maintained by other colleagues.